src/gt4py/analysis/passes.py

```diff
@@ -35,6 +35,7 @@
 )
 
 from gt4py import definitions as gt_definitions
+from gt4py import gtscript
```

Comment:
This is not used (and flake8 should actually complain about it... it's weird that github actions didn't catch it)
```suggestion
```